docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Windows_SDK.md

```diff
@@ -97,9 +97,9 @@ The minimum requirements (builds tools and SDKs, not including Windows Mobile Em
 
 ## Troubleshooting
 
-* Appcelerator Studio does not show any emulators or connected devices/`ti info -t windows` does not display any information
+* The CLI does not show any emulators or connected devices/`ti info -t windows` does not display any information
```

Comment:
I believe this entire troubleshooting section has to do with the old Windows platform which is dead, so we can remove this entire section.